app/Transformers/BeatmapsetTransformer.php

```diff
@@ -128,8 +130,32 @@ public function includeUser(Beatmapset $beatmapset)
     public function includeBeatmaps(Beatmapset $beatmapset)
     {
         return $this->collection(
-            $beatmapset->defaultBeatmaps,
+            $beatmapset->beatmaps,
             new BeatmapTransformer()
         );
     }
+
+    public function includeConverts(BeatmapSet $beatmapSet)
+    {
+        $standardBeatmaps = $beatmapSet->beatmaps()->where('playmode', 0)->get();
```

Comment:
considering `$beatmapset->beatmaps` should be already loaded, wouldn't it save a query to filter in the foreach loop?